test/per_file_coverage.sh

```diff
@@ -11,7 +11,7 @@ declare -a KNOWN_LOW_COVERAGE=(
 "source/common/event:95.1" # Emulated edge events guards don't report LCOV
 "source/common/filesystem/posix:96.3" # FileReadToEndNotReadable fails in some env; createPath can't test all failure branches.
 "source/common/http/http2:96.0"
-"source/common/json:94.8"
+"source/common/json:94.4"
```

Comment:
Usually we should not just lower our expectations, but I suspect the issue here is that the stream was previously 100% covered, and it still is. However, we lowered the overall count of lines, so the uncovered lines elsewhere in the json directory now represent a greater percentage of the total lines.

Can you verify that's correct?

Reply:
Yeah, you are right. (Although two lines code guard by NDEBUG was not covered).

I have tried to improve other coverage of other source fine. But the interface of `Json::Object` prohibit me to do it. There are branches is hard to reach. I think it would be another issue to optimize the loader code.